homeassistant/components/mfi/sensor.py

```diff
@@ -53,7 +53,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     host = config.get(CONF_HOST)
     username = config.get(CONF_USERNAME)
     password = config.get(CONF_PASSWORD)
-    use_tls = config.get(CONF_SSL)
+    use_tls = config[CONF_SSL]
```

Comment:
These tests should preferably be rewritten to set up the component and platform, to let the core do config validation:
https://github.com/home-assistant/core/blob/dev/tests/components/mfi/test_sensor.py